jj/logs/formatters/_ext_simple_formater.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import os
+from logging import LogRecord
+from string import Template
+
+from ...requests import Request
+from ...responses import Response
+from ._simple_formatter import SimpleFormatter
+
+__all__ = ("ExtSimpleFormatter",)
+
+INFO = logging.INFO
+DEBUG = logging.DEBUG
+
+log_level = int(os.getenv('MOCK_LOG_LEVEL', INFO))
+body_output_limit = int(os.getenv('MOCK_BODY_OUTPUT_LIMIT', 10 ** 6))
+format = os.getenv('MOCK_FORMAT',
```

Comment:
It appears that some of the logic in this code overlaps with what's found in `jj/logs/__init__.py`. Might I suggest passing `log_level` and `log_format` directly to the constructor as shown below? This could help streamline the code:

```python
formatter = ExtSimpleFormatter(log_level, log_format)
```